kuksa-sdk/src/main/kotlin/org/eclipse/kuksa/extension/KuksaValResponseExtension.kt

```diff
@@ -36,3 +36,18 @@ val KuksaValV1.GetResponse.metadata: Types.Metadata?
 
         return entry.metadata
     }
+
+/**
+ * Convenience property which returns the first value ([Types.Datapoint]) from the [KuksaValV1.GetResponse].
+ */
+val KuksaValV1.GetResponse.value: Types.Datapoint?
```

Comment:
maybe rename to firstValue to make it clearer that we just access the first element from the list